siteapp/urls.py

```diff
@@ -97,6 +117,11 @@
     url(r'^health/request-headers$', views_health.request_headers),
     url(r'^health/request$', views_health.request),
     url(r'^health/debug$', views.debug, name="debug"),
+
+
```

Comment:
1. maybe delete extra blank line?

2. add a "# tags" comment line to group the routes